vadin-demo-gui/src/main/java/de/muenchen/vaadin/services/BuergerServiceImpl.java

```diff
@@ -59,10 +66,9 @@ public Buerger create(Buerger buerger) {
         try {
             returnBuerger = result.get(TIMEOUT_VAL, TimeUnit.SECONDS);
             showSuccessNotification(I18nPaths.NotificationType.success, SimpleAction.create);
-        } catch (ExecutionException e) {
-            HttpClientErrorException exception = (HttpClientErrorException) e.getCause();
+        } catch (HttpClientErrorException e) {
             returnBuerger = BuergerFallbackDataGenerator.createBuergerFallback();
```

Comment:
Das hier ist falsch. Der ExecutorService wrapped auftretende exceptions zunächst in eine execution-exception, weshalb diese zunächst abgefangen werden soll.